monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/frontends.dashboard.py

```diff
@@ -0,0 +1,50 @@
+from grafanalib.core import (
+    Dashboard, Graph, Row, Target
+)
```

Comment:
Minor style thing: please use the one line syntax:
from grafanalib.core import Dashboard, Graph, Row, Target